tests/tests_sample_scenes.py

```diff
@@ -2,7 +2,11 @@
 
 # This file is intended to test any new feature added. 
 # Feel free to add a test or to modify one when adding a new/changing a feature. 
-
+class Test(Scene): 
```

Comment:
The scenes in this file are no better than the ones in example_scenes in that they must be run manually and only tell us whether or not the classes involved will crash.
Instead of this we should eventually write proper tests that can be run in a test suite.